api/v1beta1/cluster_types.go

```diff
@@ -908,6 +920,7 @@ type ClusterNetwork struct {
 type NetworkRanges struct {
 	// cidrBlocks is a list of CIDR blocks.
 	// +kubebuilder:validation:MaxItems=100
+	// +kubebuilder:validation:items:MaxLength=256
```

Comment:
A valid CIDR notation is at most 43 characters long (8 blocks of 4, 7 separators, a slash and up to 3 numeric characters for the prefix)

Reply:
I wasn't including IPv4-mapped-IPv6 addresses as Kube doesn't support them (see https://github.com/kubernetes/kubernetes/pull/121912#discussion_r1395670555), a plan IPv6 address is only 43 chars.

Reply:
Perfect, thank you!